configs/components/augeas.rb

```diff
@@ -15,7 +15,7 @@
     raise "augeas version #{version} has not been configured; Cannot continue."
   end
 
-  pkg.url "http://download.augeas.net/augeas-#{pkg.get_version}.tar.gz"
+  pkg.url "https://github.com/hercules-team/augeas/releases/download/release-#{pkg.get_version}/augeas-#{pkg.get_version}.tar.gz"
```

Comment:
Looks like this broke in 9c6456a35f3f044d3973e2e8dc341dee8eb8a72e AFAIK, only 1.13.0+ is on github

Reply:
@joshcooper fixed!